### DISCUSSION TITLE
[잡담] 과제하려고 하는데 혹시 저만 다른 분들이 짜놓은 코드가 안보이나용...?

### DISCUSSION THREAD

**Discussion by jiwoo4424** [4Moyede/HexatonClass-01#84] [잡담]
다른 분들이 짜놓은 코드들 보고 리뷰해주려고 보니깐 저한테는 00폴더만 보이면서 1/2일의 과제들이 안보이는데 다른 분들은 보이시나요???/  저만 바보인가요 ? ㅠㅠㅠㅠㅠㅠㅠ

**Comment by jiwoo4424** [comment]
어......해결했습니다 제가 바보였어요 ...... 늦은 시간에 죄송합니다....

**Comment by yoomambo** [comment]
![image](https://user-images.githubusercontent.com/54018187/104028064-9d2caa00-520b-11eb-8a71-faead788c15b.png)

멘티분들이 아직 pull request 안보내셔...앗

**Reply by yoomambo** [reply]
github 내에서 저거 클릭하면 각 멘티들 branch 들어갈수있습니당다라당당

**Reply by jiwoo4424** [reply]
감사합니당당구리

**Reply by dev-LeoKang** [reply]
앗앗
